x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/gcp_credentials_form/gcp_credential_form.tsx

```diff
@@ -47,7 +43,18 @@ import {
   GCP_CREDENTIALS_TYPE_OPTIONS_TEST_SUBJ,
 } from '../../test_subjects';
 
-type SetupFormatGCP = 'google_cloud_shell' | 'manual';
+export const GCP_SETUP_ACCESS = {
+  CLOUD_SHELL: 'google_cloud_shell',
+  MANUAL: 'manual',
+};
+
+export const GCP_CREDENTIALS_TYPE = {
+  CREDENTIALS_FILE: 'credentials-file',
+  CREDENTIALS_JSON: 'credentials-json',
+  CREDENTIALS_NONE: 'credentials-none',
+};
+
+type SetupFormatGCP = typeof GCP_SETUP_ACCESS.CLOUD_SHELL | typeof GCP_SETUP_ACCESS.MANUAL;
```

Comment:
@maxcold 👋  looks like `SetupFormatGCP` [evaluates](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAYwgOwM7wOIGEAKB9AZQFEAVAVXwEEssiCC4BeOAbwCg5O4sAZAeTIARQgAkiPHgC44AcgDmECHIA2wPAmUQArgBM8qABbBlymQBoOXALKUAcmUpTZAWwCGyLa9MWAvgG42NlBIWEQUdDhsfCwAJSJBIlsSAElHAjwSAE0cIiZWS1j4xJS0vAAxZJ4iaRkEKGAdYGQYAEsvVABaADMW1XMCuISk1J50gCkCPlsauoam1vaOgCtUFH7OQqGS0bxbKerZWcbmtuVO5BRgfv9AmABPMGA4AmAYLTAy6DcYKLz7x4gXUiuEIpAoeGotHoADpeAJhAQxBI4AAfOD-YCA4H4YjkKg0OgEaE2eyOAKBAD0VLgrlQYTQ8ECwWg8CQDOxoLxEIJ9AATHl2FxuPwhKJxE55IoVGoNNo9IZjN5LJwSQ4JW4PF5rjS6Wz0OTmaE9ZgQZtiiN0lkcvzmILuINzaUKlUZvVjgszt1elcLBsHcNShMpq65idFis1r77UUAzs9rYDrU3fNTudLtrafT9bcHk8Xm8Pl9XD9cDbOBisVFOeDIYTebCRQikTxUejc5WQbiazyCPXVWS2Bjnq93p8oN8ogBmPLl9tAqtd-FQ3sAbQA1sA7liK-PO2Cl3WALpsIA) to `string` 

we should use `as const` for both `GCP_CREDENTIALS_TYPE` and `GCP_SETUP_ACCESS`

Reply:
@orouz  great catch, thanks for bringing it up. I will prepare a PR with the fix!